docker-compose.yaml

```diff
@@ -15,8 +15,11 @@ services:
   db:
     container_name: twomes-backoffice-api-local-db
     image: mariadb:latest
-    ports:
-      - 3306:3306
```

Comment:
This docker compose file is only used for local testing, not for deploying it to the API. Because of that, and the fact that I also would like to keep it as simple as possible for anyone to run this locally, we should keep the port exposed by default.

Something we could do to possibly prevent problems with multiple database containers that have ports exposed, is to not use 3306.